### PR TITLE
add program_marketing backend ("Explore New XSeries" functionality)

### PR DESCRIPTION
**Pull request by dgamanenko**
@maxsocl 
please review)

### CODE REVIEW

File: common/templates/program_marketing/marketing_page.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<!-- <!DOCTYPE html>
```
Comment (maxsocl) [inline common/templates/program_marketing/marketing_page.html]: Remove commented code.

File: lms/templates/learner_dashboard/program_details0.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Override the default styles_version to the Pattern Library version (version 2)
```
Comment (maxsocl) [inline lms/templates/learner_dashboard/program_details0.html]: Remove this file

File: lms/templates/learner_dashboard/program_details_view0.underscore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<header class="js-program-header program-header full-width-banner"></header>
```
Comment (maxsocl) [inline lms/templates/learner_dashboard/program_details_view0.underscore]: Remove this file

File: lms/templates/program_marketing/curriculum.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<%inherit file="../main.html" />
```
Comment (maxsocl) [inline lms/templates/program_marketing/curriculum.html]: Remove this file too

**Comment by maxsocl** [comment]
LGMT when fix

File: lms/templates/learner_dashboard/program_details_view0.underscore (hunk 3 above)
Comment (dgamanenko) [inline lms/templates/learner_dashboard/program_details_view0.underscore]: done

File: lms/templates/learner_dashboard/program_details0.html (hunk 2 above)
Comment (dgamanenko) [inline lms/templates/learner_dashboard/program_details0.html]: done

**Pull request merged**